pkg/scheduler/eventhandlers.go

```
@@ -255,7 +284,7 @@ func addAllEventHandlers(
 	gvkMap map[framework.GVK]framework.ActionType,
 ) {
 	// scheduled pod cache
-	informerFactory.Core().V1().Pods().Informer().AddEventHandler(
+	resourceEventHandlerRegistration, _ := informerFactory.Core().V1().Pods().Informer().AddEventHandler(
```

Too long. `handlerRegistration` should be enough.

---

Also, do not silence the error, return it.